1.10/installing/custom/system-requirements/index.md

```diff
@@ -57,7 +57,10 @@ Here are the agent node hardware requirements.
 The agent nodes must also have: 
 
 - A `/var` directory with 10 GB or more of free space. This directory is used by the sandbox for both [Docker and DC/OS Universal container runtime](/docs/1.10/deploying-services/containerizers/).
-- Network Access to a public Docker repository or to an internal Docker registry.
+
+- The agent's work directory, `/var/lib/mesos/slave`, should be on a separate device. This protects all the other services from a task overflowing the disk. The agent does not enforce the disk resource. This may change in the future, so we recommend assigning the disk resource in your apps and jobs.
```

Comment:
Depending on how in-depth you want this blurb to be, you can mention the following:

* The disk resource is not enforced due to backwards compatibility with frameworks written before the disk resource was introduced.
* The user can turn on disk resource enforcement by inserting the environment variable `MESOS_ENFORCE_CONTAINER_DISK_QUOTA=true` into one of the Mesos agent extra config files (e.g. `/var/lib/dcos/mesos-slave-common`).
* Disk quotas are not supported by Docker tasks, so these can overflow the disk regardless of configuration.